src/ray/protobuf/usage.proto

```diff
@@ -130,10 +130,17 @@ enum TagKey {
   // Name of Tune scheduler algorithm or "Custom" if user-defined.
   // Example: "FIFOScheduler"
   TUNE_SCHEDULER = 502;
-
   // Ray AIR environment variable usage stored in JSON list format
   // This lists which of the environment variables exposed by the AIR libraries
   // are provided by the user.
   // Ex: ["RAY_AIR_LOCAL_CACHE_DIR", "TUNE_FALLBACK_TO_LATEST_CHECKPOINT"]
   AIR_ENV_VARS = 503;
+  // Fully user-controlled experiment tracking integrations ("1" if used)
+  // NOTE: These tags + the callback metrics can be aggregated to extract
+  // total experiment tracking integration usage.
+  AIR_SETUP_WANDB_INTEGRATION_USED = 504;
+  AIR_SETUP_MLFLOW_INTEGRATION_USED = 505;
```

Comment:
Do we have a guideline on when we want to add new flags vs. a collection (e.g. a list of integrations used)?